ibm/provider.go

```diff
@@ -244,6 +244,7 @@ func Provider() *schema.Provider {
 			"ibm_is_flow_logs":                       dataSourceIBMISFlowLogs(),
 			"ibm_is_image":                           dataSourceIBMISImage(),
 			"ibm_is_images":                          dataSourceIBMISImages(),
+			"ibm_is_endpoint_gateway_targets":        dataSourceIBMISEndpointGatewayTarget(),
```

Comment:
Can you please make `dataSourceIBMISEndpointGatewayTarget` ==> `dataSourceIBMISEndpointGatewayTargets`